guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -3,31 +3,50 @@
 
 Each provisioning type requires some network configuration. Use this chapter to configure network services in {ProjectServer}'s integrated {SmartProxy}.
 
-New hosts must have access to your {SmartProxyServer}. A {SmartProxyServer} can be either {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer}. You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to the {ProjectServer} directly, or when the content is synchronized with the {SmartProxyServer}. Provisioning using the external {SmartProxyServer} can save on network bandwidth.
+New hosts must have access to your {SmartProxyServer}. {SmartProxyServer} can be either {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer}. You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to the {ProjectServer} directly, or when the content is synchronized with the {SmartProxyServer}. Provisioning using the external {SmartProxyServer} can save on network bandwidth.
 
-Configuring the {SmartProxyServer} has two basic requirements:
+Configuring {SmartProxyServer} has two basic requirements:
```

Comment:
Learning articles grammer is on my todo list, particularly when there's no article ;-)

Reply:
@lzap lol! This is a terrible example and not great in English, but complies with marketing requirements for Capsule.